json-resume-builder/src/components/Education.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+import resumeData from "../resume.json";
+
+export default class Education extends Component {
+  render() {
```

Comment:
very minor, maybe look into using functional components and hooks? It seems to be where the React community is heading towards. No issues using class though.

Reply:
Yah hooks is my next thing Im going to be diving into